models/ed/R/write.configs.ed.R

```diff
@@ -102,12 +102,14 @@ convert.samples.ED <- function(trait.samples) {
 ##' @param defaults list of defaults to process. Default=settings$constants
 ##' @param check Logical. If `TRUE`, check ED2IN validity before running and 
 ##' throw an error if anything is wrong (default = `FALSE`)
-##' @param inputs updated input paths coming from SDA workflow, will modify ED2IN
+##' @param inputs updated met paths coming from SDA workflow, will modify ED2IN
+##' @param IC updated css/pss/site paths coming from SDA workflow, will modify ED2IN
+##' 
 ##' @return configuration file and ED2IN namelist for given run
 ##' @export
 ##' @author David LeBauer, Shawn Serbin, Carl Davidson, Alexey Shiklomanov
 ##-------------------------------------------------------------------------------------------------#
-write.config.ED2 <- function(trait.values, settings, run.id, defaults = settings$constants, check = FALSE, inputs = NULL, ...) {
+write.config.ED2 <- function(trait.values, settings, run.id, defaults = settings$constants, check = FALSE, inputs = NULL, IC = NULL, ...) {
```

Comment:
Do not add an IC arg to write.configs.ED2. The IC info is all coming from the settings.

Reply:
I agree, I was following the SIPNET example. I guess, I can populate the settings$inputs list and disintegrate [somewhere here](https://github.com/PecanProject/pecan/blob/develop/modules/assim.sequential/R/sda.enkf.R#L79_L111). I just thought this will change after refactoring and ensemble generator anyway

Reply:
Yeah, I think @para2x has refactored that approach away and hopefully the PR with that in place will be submitted very soon. 

The addition of an IC arg in write.configs.SIPNET started as a quick hack in the final sprint when I was trying to get SDA working for the first PalEON summer course. Deeply regret not fixing it long ago.